src/App.js

```diff
@@ -10,29 +10,34 @@ import HomePage from './shared/pages/Home';
 
 import VisitorRoutes from './visitors';
 import CbAdminRoutes from './cb_admin';
-
-
-export default class App extends Component {
-
-  componentWillMount() {
-    // avoid eslint errors to keep this a class component
-  }
-
-  render() {
-    return (
-      <Dots>
-        <Container>
-          <Switch>
-            <Route exact path="/" component={HomePage} />
-            <Route path="/visitor/*" component={VisitorRoutes} />
-            <Route path="/cb/*" component={CbAdminRoutes} />
-
-            <Route exact path="/error/404" component={NotFound} />
-            <Route exact path="/error/:code" component={InternalServerError} />
-            <Route component={NotFound} />
-          </Switch>
-        </Container>
-      </Dots>
-    );
-  }
-}
+import Login from './cb_admin/pages/Login';
+import PrivateCookieRoute from './auth/PrivateCookieRoute';
```

Comment:
Can this be renamed to something more generic like `PrivateRoute`. "Cookie" refers to an implementation detail that is not relevant to the fundamental purpose of the component. Allowing implementation details to affect naming means at some point in the future you end up with names that don't quite make sense.

Reply:
Yeah, it’s left over from when I was going to write a HOC for the admin section. Not relevant anymore.